topics/icrc_25_signer_interaction_standard.md

```diff
@@ -297,10 +267,7 @@ Request
 {
     "id": 1,
     "jsonrpc": "2.0",
-    "method": "icrc25_granted_permissions",
-    "params": {
-        "version": "1"
-    }
+    "method": "icrc25_granted_permissions"
```

Comment:
This example (and other examples below) still uses "Response" (line 274) rather than "Result".

Reply:
Ah, I see.  Sorry for the noise.